packages/core/test/card/cardTests.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import { assert } from "chai";
+import { shallow } from "enzyme";
+import * as React from "react";
+
+import { Card, Classes } from "../../src/index";
+
+describe("<Card>", () => {
```

Comment:
These are not very useful tests and IMO are a good example of how unit test coverage thresholds can be arbitrary. The maintenance + performance cost of these tests isn't worth what they provide us. Let's remove them + use `/* istanbul ignore next */` directives and/or lower the coverage thresholds.